Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1041,9 +993,6 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             }
 
             settings.selectedServer = .endpoint(serverName)
-            if case .connected = connectionStatus {
-                try? await updateTunnelConfiguration(environment: settings.selectedEnvironment, serverSelectionMethod: .preferredServer(serverName: serverName), reassert: true)
```

Comment:
This change is actually changing behaviour.  Why is it no longer necessary to update the tunnel configuration when the server selection changes?

Reply:
We still do. This now handles two cases: https://github.com/duckduckgo/BrowserServicesKit/pull/764/files#r1559075769